config/204-istio-default-domain.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: default-domain
+  name: default-domain
+  namespace: knative-serving
+  labels:
+    app: "default-domain"
```

Comment:
Perhaps we should include labels on istio things so folks can install without istio via `kubectl apply -f release.yaml -l foo!=istio` 🤔

Reply:
Done